Dockerfile

```diff
@@ -14,7 +14,20 @@ RUN apt-get update && \
         cd xml-doclet/; mvn install && \
         cd .. ; rm -rf xml-doclet && \
     sed 's/.ALL:ALL./(ALL) NOPASSWD:/' -i /etc/sudoers 
+RUN until apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv EF678DA6D5B1436D3972DFD317BE949418BE5D6B; do echo retrying; done && \
```

Comment:
I moved back to the top because this step to add the repo that contains zenta never changes, so it can together with steps that don't change frequently.

Reply:
You will need and apt-get update before installing zenta stuff.
The repo location and key won't change frequently, but the content will.

Reply:
But even an `apt-get update` could be cached and the new version would not be installed.
As I mentioned before, best solution is specify the version of zenta and zenta-tools we want to install. This way we have traceability for which version of zenta we-re using and we also only download a new version when there's in fact a new version.
Downside is every time a new version of zenta is released we need to update the Dockerfile. But it's not so frequent. And traceability of version of zenta we're using is a very good practice.